### ISSUE TITLE
WP Rocket configuration conflict: An error occurred while running 'mapSelect': Cannot read properties of null (reading 'isGatheringData')

### ISSUE THREAD

**Issue by jamesozzie** [google/site-kit-wp#4449]
## Bug Description

There have been reports recently of the below error appearing when checking the Site Kit tab with a particular WP Rocket configuration:
`An error occurred while running 'mapSelect': Cannot read properties of null (reading 'isGatheringData')
`

This has been reproducible in support and is as a result of a [particular set of WP Rocket configurations](https://i.imgur.com/DmQTFmC.png) (File Optimization > JavaScript files) with caching for logged in users enabled. 

```
An error occurred while running 'mapSelect': Cannot read properties of null (reading 'isGatheringData')
The error may be correlated with this previous error:
TypeError: Cannot read properties of null (reading 'isGatheringData')
    at Object.current (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-adminbar-c258be922e5623a631c3.js?ver=1638479187:2:18144)
    at e (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:231724)
    at https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:231833
    at Ui (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:811566)
    at ya (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:834493)
    at t.unstable_runWithPriority (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:854118)
    at Mr (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:779218)
    at Yj (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:830920)
    at Lj (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:821278)
    at https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:779509

Original stack trace:

    in AdminBarSessions
    in WithWidgetComponentProps(AdminBarSessions)
    in div
    in Cell
    in div
    in Row
    in AdminBarWidgets
    in div
    in div
    in div
    in AdminBarApp
    in RestoreSnapshots
    in ErrorHandler
    in Root
````

For those impacted the following workarounds can be used:
- Disable caching for logged in users ([screenshot](https://i.imgur.com/erqQvX5.png))
- Apply an exclusion for Site Kit files - `/wp-content/plugins/google-site-kit/dist/assets/js/(.*).js`  - _File Optimization > JavaScript Files > Excluded JavaScript Files_ ([screenshot](https://i.imgur.com/CviFCyd.png))
- Install and activate [a mini plugin](https://gist.github.com/adamsilverstein/9933f3efb479544e6f8a8f026ddf58ca) that excludes Site Kit resources from minification

<details><summary>Site Health information from support testing site</summary>

```
### wp-core ###

version: 5.8.2
site_language: en_US
user_language: en_US
permalink: /%post_id%/%postname%/
https_status: true
user_registration: 0
default_comment_status: closed
multisite: false
user_count: 4
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /var/www/vhosts/plastiskip.com/httpdocs
wordpress_size: 103.60 MB (108631880 bytes)
uploads_path: /var/www/vhosts/plastiskip.com/httpdocs/wp-content/uploads
uploads_size: 6.06 MB (6358276 bytes)
themes_path: /var/www/vhosts/plastiskip.com/httpdocs/wp-content/themes
themes_size: 20.52 MB (21513999 bytes)
plugins_path: /var/www/vhosts/plastiskip.com/httpdocs/wp-content/plugins
plugins_size: 77.68 MB (81454694 bytes)
database_size: 41.69 MB (43715008 bytes)
total_size: 249.55 MB (261673857 bytes)

### wp-dropins (2) ###

advanced-cache.php: true
maintenance.php: true

### wp-active-theme ###

name: Twenty Twelve (twentytwelve)
version: 3.5
author: the WordPress team
author_website: https://wordpress.org/
parent_theme: none
theme_features: core-block-patterns, widgets-block-editor, editor-style, editor-styles, wp-block-styles, responsive-embeds, editor-color-palette, automatic-feed-links, post-formats, menus, custom-background, post-thumbnails, customize-selective-refresh-widgets, custom-header, amp, widgets
theme_path: /var/www/vhosts/plastiskip.com/httpdocs/wp-content/themes/twentytwelve

### wp-themes-inactive (7) ###

AMPFace: version: 1.6.0, author: James Ozzie Osborne
Astra: version: 3.2.0, author: Brainstorm Force (latest version: 3.7.5)
The7: version: 9.16.0, author: Dream-Theme
NewsCard: version: 1.2.7, author: Theme Horse (latest version: 1.2.8)
Startup Shop: version: 1.0.2, author: aThemeArt (latest version: 1.0.3)
Twenty Twenty: version: 1.7, author: the WordPress team (latest version: 1.8)
Wellington: version: 2.1.4, author: ThemeZee

### wp-mu-plugins (1) ###

Health Check Troubleshooting Mode: author: (undefined), version: 1.7.2

### wp-plugins-active (26) ###

AMP: version: 2.1.4, author: AMP Project Contributors
Contact Form 7: version: 5.5.3, author: Takayuki Miyoshi
Debug: version: 1.9, author: SoniNow Team
Debug This: version: 0.6.3, author: WPChill
Elementor: version: 3.4.8, author: Elementor.com
ElementsKit Lite: version: 2.5.0, author: Wpmet
Email Address Encoder: version: 1.0.22, author: Till Krüss
GDPR Cookie Consent: version: 2.0.6, author: WebToffee
Header Footer Code Manager: version: 1.1.15, author: 99robots
Health Check & Troubleshooting: version: 1.4.5, author: The WordPress.org community
Kirki Customizer Framework: version: 3.1.9, author: David Vongries
MC4WP: Mailchimp for WordPress: version: 4.8.6, author: ibericode
MetForm: version: 2.0.0, author: Wpmet
Mighty Addons: version: 1.7.0, author: MightyThemes
REST API Log: version: 1.6.8, author: Pete Nelson
Salert: version: 1.1.9, author: WPoperation
Site Kit by Google: version: 1.46.0, author: Google
Site Kit by Google Tester: version: 1.6.0, author: Google
Tidio Chat: version: 4.5.1, author: Tidio Ltd.
Unyson: version: 2.7.24, author: ThemeFuse
WP File Manager: version: 7.1.2, author: mndpsingh287
WPForms Lite: version: 1.7.1.2, author: WPForms
WP Rocket: version: 3.8.8, author: WP Media
Wp Social: version: 1.8.2, author: Wpmet
Wp Ultimate Review: version: 1.4.3, author: Wpmet
Yoast Duplicate Post: version: 4.2, author: Enrico Battocchi & Team Yoast

### wp-plugins-inactive (40) ###

A2 Optimized WP: version: 2.1.3.5, author: A2 Hosting (latest version: 2.1.3.8)
Akismet Anti-Spam: version: 4.1.9, author: Automattic (latest version: 4.2.1)
AMP-position-observer Back to Top: version: 1.0, author: James Osborne
AMP Generic Compat: version: 0.1, author: milindmore22
AMP Infinite Scroll: version: 0.1, author: milindmore22, Google
AMP Logo Rendering: version: 0.1, author: James Osborne
Asset CleanUp: Page Speed Booster: version: 1.3.8.4, author: Gabe Livan
Autoptimize: version: 2.9.3, author: Frank Goossens (futtta)
CSP Manager: version: 1.1.0, author: Patrick Sletvold
Customer Reviews for WooCommerce: version: 4.23, author: CusRev (latest version: 4.31)
Custom Post Type UI: version: 1.9.2, author: WebDevStudios (latest version: 1.10.0)
Duplicator: version: 1.4.2, author: Snap Creek (latest version: 1.4.3)
Easy Image Collage: version: 1.13.2, author: Bootstrapped Ventures
EWWW Image Optimizer: version: 6.2.4, author: Exactly WWW (latest version: 6.3.0)
Feature Box: version: 1.0.1, author: Sumaiya Siddika
Featured Image from URL (FIFU): version: 3.7.0, author: fifu.app (latest version: 3.8.0)
iThemes Security: version: 8.0.1, author: iThemes (latest version: 8.0.2)
Jetpack by WordPress.com: version: 9.5, author: Automattic (latest version: 10.3)
LiteSpeed Cache: version: 4.4.3, author: LiteSpeed Technologies (latest version: 4.4.5)
Loco Translate: version: 2.5.3, author: Tim Whitlock (latest version: 2.5.5)
podcast player: version: 5.0.1, author: vedathemes (latest version: 5.4.0)
Premmerce: version: 1.3.15, author: premmerce
Regenerate Thumbnails: version: 3.1.5, author: Alex Mills (Viper007Bond)
SaFly Curl Patch: version: 1.0.0, author: SaFly Organization
Salt Shaker: version: 1.2.8, author: Nagdy (latest version: 1.2.9)
SiteKitWPRocketCompat: version: 1.0.0, author: Adam Silverstein, Google
Ultimate Addons for Gutenberg: version: 1.25.0, author: Brainstorm Force (latest version: 1.25.2)
User Profile Picture: version: 2.6.0, author: Cozmoslabs
Visualizer: Tables and Charts for WordPress: version: 3.7.0, author: Themeisle (latest version: 3.7.1)
W3 Total Cache: version: 2.1.9, author: BoldGrid (latest version: 2.2.0)
Webcraftic Clearfy – WordPress optimization plugin: version: 1.9.4, author: Creative Motion <info@cm-wp.com> (latest version: 2.0.0)
Web Stories: version: 1.7.2, author: Google (latest version: 1.15.0)
WooCommerce: version: 5.6.0, author: Automattic (latest version: 5.9.0)
WP-Optimize - Clean, Compress, Cache: version: 3.1.12, author: David Anderson, Ruhani Rabin, Team Updraft (latest version: 3.2.1)
WP-Optimize Premium - Clean, Compress, Cache: version: 3.1.8, author: David Anderson, Ruhani Rabin, Team Updraft
WP-REST-API V2 Menus: version: 0.10, author: Claudio La Barbera
WPBakery Page Builder: version: 6.6.0, author: Michael M - WPBakery.com
WP Fastest Cache: version: 0.9.5, author: Emre Vona
WP Mobile Redirect: version: 2.6, author: Iqbal Bary <contact@iqbalbary.com>
WP Umbrella: version: 1.6.0, author: WP Umbrella (latest version: 1.8.1)

### wp-media ###

image_editor: WP_Image_Editor_Imagick
imagick_module_version: 1690
imagemagick_version: ImageMagick 6.9.10-68 Q16 x86_64 2021-10-14 https://imagemagick.org
imagick_limits: 
	imagick::RESOURCETYPE_AREA: 11 GB
	imagick::RESOURCETYPE_DISK: 9.2233720368548E+18
	imagick::RESOURCETYPE_FILE: 1536
	imagick::RESOURCETYPE_MAP: 11 GB
	imagick::RESOURCETYPE_MEMORY: 6 GB
	imagick::RESOURCETYPE_THREAD: 1
gd_version: bundled (2.1.0 compatible)
ghostscript_version: 9.25

### wp-server ###

server_architecture: Linux 3.10.0-957.5.1.el7.x86_64 x86_64
httpd_software: Apache
php_version: 7.4.26 64bit
php_sapi: cgi-fcgi
max_input_variables: 1000
time_limit: 60
memory_limit: 256M
max_input_time: 60
upload_max_size: 16M
php_post_max_size: 16M
curl_version: 7.29.0 NSS/3.53.1
suhosin: false
imagick_availability: true
server-headers: unknown
htaccess_extra_rules: true

### wp-database ###

extension: mysqli
server_version: 5.5.68-MariaDB
client_version: mysqlnd 7.4.26

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /var/www/vhosts/plastiskip.com/httpdocs/wp-content
WP_PLUGIN_DIR: /var/www/vhosts/plastiskip.com/httpdocs/wp-content/plugins
WP_MAX_MEMORY_LIMIT: 256M
WP_DEBUG: true
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: true
SCRIPT_DEBUG: true
WP_CACHE: true
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_LOCAL_DEV: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable
mu-plugins: writable

### wpforms ###

version: 1.7.1.2
lite: Dec 2, 2021 @ 8:50pm
upload_dir: Writable
db_tables: wp_wpforms_tasks_meta
total_forms: undefined
total_submissions: undefined

### google-site-kit ###

version: 1.46.0
php_version: 7.4.26
wp_version: 5.8.2
reference_url: https://plastiskip.com
amp_mode: secondary
site_status: connected-site
user_status: authenticated
verification_status: verified-non-site-kit
connected_user_count: 3
active_modules: site-verification, search-console, adsense, analytics, analytics-4, pagespeed-insights, tagmanager, idea-hub
required_scopes: 
	openid: ✅
	https://www.googleapis.com/auth/userinfo.profile: ✅
	https://www.googleapis.com/auth/userinfo.email: ✅
	https://www.googleapis.com/auth/siteverification: ✅
	https://www.googleapis.com/auth/webmasters: ✅
	https://www.googleapis.com/auth/adsense.readonly: ✅
	https://www.googleapis.com/auth/analytics.readonly: ✅
	https://www.googleapis.com/auth/tagmanager.readonly: ✅
	https://www.googleapis.com/auth/ideahub.read: ✅
	https://www.googleapis.com/auth/ideahub.full: ✅
capabilities: 
	googlesitekit_authenticate: ✅
	googlesitekit_setup: ✅
	googlesitekit_view_posts_insights: ✅
	googlesitekit_view_dashboard: ✅
	googlesitekit_view_module_details: ✅
	googlesitekit_manage_options: ✅
enabled_features: 
	ideaHubModule: ✅
	serviceSetupV2: ✅
	swgModule: ⭕
	userInput: ⭕
	unifiedDashboard: ⭕
search_console_property: https://plastiskip.com/
adsense_account_id: none
adsense_client_id: none
adsense_account_status: none
adsense_use_snippet: yes
adsense_web_stories_adunit_id: none
analytics_account_id: 1838•••••
analytics_property_id: UA-1838•••••••
analytics_profile_id: 2335•••••
analytics_use_snippet: yes
analytics_4_property_id: 2685049••
analytics_4_web_data_stream_id: 2417••••••
analytics_4_measurement_id: G-JT••••••••
analytics_4_use_snippet: yes
tagmanager_account_id: none
tagmanager_container_id: none
tagmanager_amp_container_id: none
tagmanager_use_snippet: yes

### amp_wp ###

amp_slug_query_var: amp
amp_slug_defined_late: false
amp_mode_enabled: transitional
amp_reader_theme: legacy
amp_templates_enabled: post, page, attachment, is_singular, is_home, is_archive, is_author, is_date, is_search, is_404, is_category, is_tag, is_post_type_archive[xs_review]
amp_serve_all_templates: true
amp_css_transient_caching_disabled: false
amp_css_transient_caching_threshold: 5000 transients per day
amp_css_transient_caching_sampling_range: 14 days
amp_css_transient_caching_transient_count: 64
amp_css_transient_caching_time_series: 
	20211119: 61
	20211120: 46
	20211121: 48
	20211122: 45
	20211123: 45
	20211124: 47
	20211125: 47
	20211126: 38
	20211127: 34
	20211128: 34
	20211129: 34
	20211130: 38
	20211201: 34
	20211202: 42
amp_libxml_version: 2.9.1
```

</details>

<details><summary>Console Errors</summary>

![image](https://user-images.githubusercontent.com/41326532/144504080-8a3e18c0-a067-4248-b36a-d4243bace195.png)

```
Caught an error: Error: An error occurred while running 'mapSelect': Cannot read properties of null (reading 'isGatheringData')
The error may be correlated with this previous error:
TypeError: Cannot read properties of null (reading 'isGatheringData')
    at Object.current (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-adminbar-c258be922e5623a631c3.js?ver=1638479187:2:18144)
    at e (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:231724)
    at https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:231833
    at Ui (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:811566)
    at ya (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:834493)
    at t.unstable_runWithPriority (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:854118)
    at Mr (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:779218)
    at Yj (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:830920)
    at Lj (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:821278)
    at https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:779509

Original stack trace:
    at Ke (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:231533)
    at AdminBarSessions (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-adminbar-c258be922e5623a631c3.js?ver=1638479187:2:18116)
    at Ih (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:793113)
    at ai (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:801335)
    at wi (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:839384)
    at ga (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:823810)
    at Zj (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:823735)
    at Lj (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:821079)
    at https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:779509
    at t.unstable_runWithPriority (https://plastiskip.com/wp-content/cache/min/1/wp-content/plugins/google-site-kit/dist/assets/js/googlesitekit-vendor-1812835601d6f08bab78.js?ver=1638479186:1:854118) ObjectcomponentStack: "\n    in AdminBarSessions\n    in WithWidgetComponentProps(AdminBarSessions)\n    in div\n    in Cell\n    in div\n    in Row\n    in AdminBarWidgets\n    in div\n    in div\n    in div\n    in AdminBarApp\n    in RestoreSnapshots\n    in ErrorHandler\n    in Root"[[Prototype]]: Object
value @ googlesitekit-adminbar-c258be922e5623a631c3.js?ver=1638479187:1
```

</details>

Impacted users:
- https://wordpress.org/support/topic/plugin-site-kit-ha-detectado-error-en-una-entrada-de-mi-blog/
- https://wordpress.org/support/topic/reporting-error/
- https://wordpress.org/support/topic/whats-happend-please/
- https://wordpress.org/support/topic/kit-del-sitio-de-google-analytics-search-console-adsense-speed/

## Steps to reproduce

1. Install and activate WP Rocket
2. Ensure caching for logged in users is enabled (Cache tab)
3. Under the "File Optimisation" tab enable "Minify JavaScript files", "Load JavaScript deferred" and "Delay JavaScript execution"
4. Visit the Site Kit admin toolbar

## Screenshots

![image](https://user-images.githubusercontent.com/41326532/144507697-7f027b44-1246-40ed-b88b-8b88e3f898ad.png)

## Additional Context

 - Reports of this started to occur only since SK 1.46.0
 - WP Rocket  3.8.8 

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by jamesozzie** [comment]
Just an additional note here:
Of the 4 impacted users - all since Site Kit 1.46.0 there are 3 different optimisation plugins referenced, so this isn't limited to WP Rocket's minification. Plugins referenced below:
- https://wordpress.org/support/topic/plugin-site-kit-ha-detectado-error-en-una-entrada-de-mi-blog/ - **Autoptimize**
- https://wordpress.org/support/topic/reporting-error - **WP Rocket**
- https://wordpress.org/support/topic/whats-happend-please - **WP Rocket**
- https://wordpress.org/support/topic/kit-del-sitio-de-google-analytics-search-console-adsense-speed - **WP Optimize**

This error is similar to the below:
https://github.com/google/site-kit-wp/issues/2350

It may be addressed with any changes to https://github.com/google/site-kit-wp/issues/4337

**Comment by aaemnnosttv** [comment]
Looks like this will be fixed for WP Rocket in an upcoming release 🎉 
See https://github.com/wp-media/wp-rocket/pull/4657

**Comment by aaemnnosttv** [comment]
@jamesozzie let's keep an eye on this PR linked above and test this when it's released (unless you're able to test it ahead of time, that would be great). Assigning to you for now but feel free to pass it to someone else so we don't forget to follow up on this. This issue won't be going through the board though it seems so this is mostly staying open for tracking.

**Comment by aaemnnosttv** [comment]
@jamesozzie can you confirm this is no longer an issue with the latest WP Rocket?

**Comment by jamesozzie** [comment]
Tested today and the issue remains @aaemnnosttv 
- SK 1.18.0
- WPRocket Version 3.10.6

The same occurs after clearing the cache and purging particular URLs. 


https://user-images.githubusercontent.com/41326532/179029763-aba636e8-3667-4a98-8001-1bd0163e0651.mp4

**Comment by aaemnnosttv** [comment]
@jamesozzie I wouldn't expect it to be fixed on WP Rocket 3.10.6 if that's what you're on. The PR mentioned above was introduced in 3.10.8 and the [latest release](https://github.com/wp-media/wp-rocket/releases) is currently 3.11.4.2. Would you please try again with a newer version?

**Comment by jamesozzie** [comment]
I can confirm this has been resolved with the current version of WP Rocket. [Recording of issue, before then updating to the latest WP Rocket version](https://recordit.co/rKz6Bfu9M6). 

Closing
